app/src/js/components/Operations/overview.js

```diff
@@ -27,38 +22,14 @@ import { operationTypes } from '../../utils/type';
 class OperationOverview extends React.Component {
   constructor(props) {
     super(props);
-    this.queryMeta = this.queryMeta.bind(this);
     this.generateQuery = this.generateQuery.bind(this);
     this.searchOperations = this.searchOperations.bind(this);
   }
 
-  componentDidMount() {
-    this.queryMeta();
-    this.props.dispatch(getCumulusInstanceMetadata());
-  }
-
   generateQuery() {
     return { ...this.props.queryParams };
   }
 
-  queryMeta() {
-    const { dispatch, queryParams } = this.props;
-    dispatch(
-      listCollections({
-        limit: 100,
-        fields: 'name,version',
-      })
-    );
-    dispatch(listWorkflows());
-    dispatch(
-      getCount({
-        type: 'executions',
-        field: 'status',
-        ...queryParams,
-      })
-    );
-  }
```

Comment:
You'd mentioned these are redundant queries, which seems very plausible. Are there Cypress tests and validate the page is responding correctly without these?

Reply:
Added cypress/integration/operations_spec.js